Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -194,10 +194,18 @@ public class SchemeGenerator {
         let launchVariables = scheme.run.flatMap { $0.environmentVariables.isEmpty ? nil : $0.environmentVariables }
         let profileVariables = scheme.profile.flatMap { $0.environmentVariables.isEmpty ? nil : $0.environmentVariables }
 
+        var testPlans: [XCScheme.TestPlanReference]?
```

Comment:
You can use `let`

```swift
let testPlanns: [XCScheme.TestPlanReference] = scheme.test?.testPlans.map { plans in
    return plans.enumerated().map { index, path in
        .init(reference: "container:\(path)", default: index == 0)
    }
}
```

Reply:
Yeah there’s a few ways to do it. The property has to be nil if there are no test plans though, as that’s what XcodeProj expects

Reply:
Points of my comment was avoiding using mutable variables as we can. 
Type should be `[XCScheme.TestPlanReference]?` Sorry.